### ISSUE TITLE
Remove Storage Swagger Readme Transforms

### ISSUE THREAD

**Issue by kurtzeborn** [Azure/azure-sdk-for-net#9054]
During the push to GA track 2, several transforms were added to the readme which need to be migrated to the swagger in azure-rest-api-specs.  This should be coordinated across all of the langauge swaggers.  
- Apply any bugfixes (ie. batch return code)
- For client side renames, we should try to use `x-ms-client-name`
- (.Net specific) Move `x-az-` values directly into the swagger, since these won't affect other languages

Once this is done, there should be no diffs in the generated code.

**Comment by JimSuplizio** [comment]
Hi @kurtzeborn, we deeply appreciate your input into this project. Regrettably, this issue has remained inactive for over 2 years, leading us to the decision to close it. We've implemented this policy to maintain the relevance of our issue queue and facilitate easier navigation for new contributors. If you still believe this topic requires attention, please feel free to create a new issue, referencing this one. Thank you for your understanding and ongoing support.
